### PR TITLE
feat: add more icons to components

### PR DESCRIPTION
**Pull request by amritk**

### CODE REVIEW

File: packages/components/src/components/ScalarIcon/icons/ArrowLeft.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="12" viewBox="0 0 14 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (antlio) [inline packages/components/src/components/ScalarIcon/icons/ArrowLeft.svg:1]: what do you think about removing the width / height attributes [as highlighted here](https://github.com/scalar/scalar/tree/main/packages/components/src/components/ScalarIcon#adding-icons-to-scalaricons) in order to preserve icon responsiveness?

Comment (amritk) [reply packages/components/src/components/ScalarIcon/icons/ArrowLeft.svg:1]: oh wow I didn't see that, yea can do.

Comment (amritk) [reply packages/components/src/components/ScalarIcon/icons/ArrowLeft.svg]: Alright done!

**Review by marclave — APPROVED** [review]
🔥 🔥 🔥 🔥 🔥 🔥

**Pull request merged**